chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+namespaces="${NAMESPACES}"
+timeout="${TIMEOUT}"
+ignoreTimeoutError="${IGNORETIMEOUTERROR}"
+
+if [ -z "${namespaces}" ]; then
+  echo "No namespace is provided."
+  exit 1
+fi
+
+if [ -z "${timeout}" ]; then
+  echo "No timeout value is provided."
+  exit 1
+fi
+
+if [ -z "${ignoreTimeoutError}" ]; then
+  echo "No ignoreTimeoutError value is provided."
+  exit 1
+fi
+
+succeeded=()
+failed=()
+
+get_pod_count() {
+  kubectl get pods --selector app="${1}" -n "${2}" -o json | jq '.items | length'
+}
+
+echo "Uninstalling Rancher resources in the following namespaces: ${namespaces}"
+
+for namespace in ${namespaces}; do
+  for app in $(helm list -n "${namespace}" -q); do
+    if [[ ${app} =~ .crd$ ]]; then
+      echo "--- Skip the app [${app}] in the namespace [${namespace}]"
+      continue
+    fi
+    echo "--- Deleting the app [${app}] in the namespace [${namespace}]"
+    if [[ ! $(helm uninstall "${app}" -n "${namespace}") ]]; then
```

Comment:
Check if this works:

```suggestion
    if ! helm uninstall "${app}" -n "${namespace}"; then
```

Reply:
it works with or without the double brackets, but I'd prefer to keep the brackets to keep the style consistent.